src/perturbed/multiplicative.jl

```diff
@@ -28,12 +36,22 @@ end
 Shorter constructor with defaults.
 """
 function PerturbedMultiplicative(
-    maximizer; ε=1.0, epsilon=nothing, nb_samples=1, rng=MersenneTwister(0), seed=nothing
-)
+    maximizer::F;
+    ε=1.0,
+    epsilon=nothing,
+    nb_samples=1,
+    rng::R=MersenneTwister(0),
+    seed::S=nothing,
+    is_parallel=false,
```

Comment:
Beware, this constructor is type unstable. Does it affect efficiency in the tests?

Reply:
To be clear, I don't think it matters, but I'd still be curious

Reply:
It doesn't seem to have changed tests duration

Reply:
how about profiling?

Reply:
Not sure this is necessary if the flame graph remains blue. Let's keep it simple